source/includes/webhooks/_department_deleted.md

```diff
@@ -0,0 +1,28 @@
+## Department deleted
+
+This web hook fires when departments are deleted from the Greenhouse system from the Configure &raquo; Organization screen. This only happens when the "Remove" button is clicked after clicking the "X" next to an office name. This will not fire individually when an organization is deleted.
+
+```json
+{
+  "action": "department_deleted",
+  "payload": {
+    "department": {
+      "id": 106,
+      "name": "Creative",
+      "child_ids": [
+        23328
+      ],
+      "parent_id": 5
+    }
+  }
+}
+```
+
+### Noteworthy response attributes
+
+| Attribute | Note |
+|------------|--------|
+| `id` | The Greenhouse id for the department |
+| `name` | The name of the department deleted |
+| `child_ids` | Any former child department ids |
```

Comment:
I'd clarify this includes all descendants, including grandchildren, etc.  Also, what this looks like if there are no children... `nil` vs `[]`